src/main/java/Duke.java

```diff
@@ -5,114 +5,167 @@
 
 public class Duke {
 
+    public static final String GREETING_MESSAGE =
+            "Hello! I'm Duke" + "\n"
+            + "What can I do for you?" + "\n"
+            + "Tell me your plan!";
+    public static final String LIST_DISPLAY_MESSAGE = "Here are the tasks in your list:";
+    public static final String MESSAGE_RECEIVED_MESSAGE = "Got it. I've added this task:";
+    public static final String SPECIFY_TIME_WARNING = "Please specify time!";
+    public static final String ILLEGAL_CMD_WARNING = "No such command! Please try again!";
+    public static final String BYE_MESSAGE = "Bye. Hope to see you again soon!";
     public static final String HORIZONTAL_LINE = "---------------------";
-    public static final String DONE_CMD = "done ";
-    public static final String BYE_CMD = "bye";
-    public static final String TODO_CMD = "todo ";
-    public static final String DEADLINE_CMD = "deadline ";
-    public static final String EVENT_CMD = "event ";
-    public static final String ILLEGAL_CMD_WARNING = "No such command! Please try again";
-    public static final String LIST_CMD = "list";
     public static final String DDL_CMD_SEPARATOR = "/by";
     public static final String EVENT_CMD_SEPARATOR = "/at";
+    public static final String DONE_CMD = "done";
+    public static final String BYE_CMD = "bye";
+    public static final String TODO_CMD = "todo";
+    public static final String DEADLINE_CMD = "deadline";
+    public static final String EVENT_CMD = "event";
+    public static final String LIST_CMD = "list";
 
     public static void main(String[] args) {
+
         greet();
         ArrayList<Task> storedTasks = new ArrayList<>();
         Scanner in = new Scanner(System.in);
-        System.out.println("Tell me your plan!");
         String response = in.nextLine();
 
-        while(true){
-            if (response.equals(BYE_CMD)) {
-                break;
-            } else if (response.startsWith(DONE_CMD) &&
-                    Character.isDigit(response.charAt(5))) {
-                int taskNumber = Integer.parseInt(response.split(" ", 2)[1]);
+        while(!response.equals(BYE_CMD)){
```

Comment:
I like the way you refactor the things in the while loop, which makes it clear for me to understand

Reply:
HAHAHA